x/storage/keeper/cross_app_group.go

```diff
@@ -374,6 +366,40 @@ func (app *GroupApp) handleUpdateGroupMemberSynPackage(ctx sdk.Context, header *
 	}
 }
 
+func (app *GroupApp) handleAddOrDeleteGroupMemberOperation(ctx sdk.Context, groupInfo *types.GroupInfo, updateGroupPackage *types.UpdateGroupMemberSynPackage) error {
+	options := types.UpdateGroupMemberOptions{
+		SourceType: types.SOURCE_TYPE_BSC_CROSS_CHAIN,
+	}
+	if updateGroupPackage.OperationType == types.OperationAddGroupMember {
+		options.MembersToAdd = updateGroupPackage.GetMembers()
+		options.MembersExpirationToAdd = updateGroupPackage.GetMemberExpiration()
```

Comment:
we need ensure the length of `MembersToAdd` and `MembersExpirationToAdd` are the same.

Reply:
we had check function in `ValidateBasic` of `UpdateGroupMemberSynPackage`

https://github.com/j75689/greenfield/blob/3b2ed010df9480f1690591e19b3f510d3655d503/x/storage/types/crosschain.go#L968